models/prices/prices_usd_latest_old.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+        schema='prices',
+        alias = alias('usd_latest_old'),
```

Comment:
nit: can we change `old` to `historical` on both file name / spell table name? the `_old` on a file / table make me think "this isn't a relevant table anymore"